src/Integration.Vsix/CFamily/CFamilyIssueToAnalysisIssueConverter.cs

```diff
@@ -63,10 +102,14 @@ public CFamilyIssueToAnalysisIssueConverter(ITextDocumentFactoryService textDocu
             this.lineHashCalculator = lineHashCalculator;
             this.fileSystem = fileSystem;
             filesContentType = contentTypeRegistryService.UnknownContentType;
+
+            pathToTextDocMap = new Dictionary<string, ITextDocument>(StringComparer.OrdinalIgnoreCase);
```

Comment:
The subprocess should use a consistent case for the file paths it returns. However, we tolerate inconsistent casing.

Reply:
It's good because I believe the subprocess is actually inconsistent between the main message and the secondary locations. I vaguely remember a bug handling that.